renovate.json

```diff
@@ -28,6 +28,17 @@
       "depNameTemplate": "com.google.protobuf:protobuf-java",
       "datasourceTemplate": "maven"
     },
+    {
+      "customType": "regex",
+      "fileMatch": [
+		"^\\.cloudbuild/library_generation/library_generation\\.Dockerfile$"
```

Comment:
Do you verify this setting will work?

Do we need to specify `includePaths`?

Reply:
@JoeWang1127 I verified it worked locally. I added the results to the PR description.

Reply:
Sounds good. Both `protobuf-java` and `protoc` are disabled:

https://github.com/googleapis/sdk-platform-java/blob/eb48f452c4c01e99d62aeeb99d51e19efbc55650/renovate.json#L154-L161

I guess we can keep these regexes and enable them afterwards?